iree-requirements-ci.txt

```diff
@@ -6,5 +6,5 @@
 # Uncomment to select a nightly version.
 # --find-links https://iree.dev/pip-release-links.html
```

Comment:
Would this work?
```
iree-base-compiler~=2.9.0
iree-base-runtime~=2.9.0
```

May also want to mention `--no-index` to use _only_ a nightly version... hmm

Reply:
No :( 
> $ uv pip install -f https://iree.dev/pip-release-links.html iree-base-runtime~=2.9.0
>   × No solution found when resolving dependencies:
  ╰─▶ Because only iree-base-runtime<=2.9.0rc20241108 is available and you require iree-base-runtime>=2.9.0,<2.10.dev0, we can conclude that your requirements are unsatisfiable.